### PR TITLE
feat: dashboards

### PR DESCRIPTION
**Pull request by 39bytes**

### CODE REVIEW

**Review by LatentDream — CHANGES_REQUESTED** [review]
## Product Testing feedback
- [x] TBD, the home page of a workspace: I think having `Product Lines | Inventory | Dashboard | Settings` on the same line / format would be better than the button.
  - Maybe workspace Settings could go in the dropdown when we click on the user profile ? ~ Might lead to confusion between global and workspace settings tho.
- Dashboard page
    - [x] Remove the `bruh` from the generated data.
    - [x] Add breadcrumb to easily understand that we are in the dashboard.
    - [x] Add sessions in generated data so the sessions overtime graph is not empty.
- Production Line page:
    - [ ] (Can be done later) The CSS overflows with half screen view.
    - [ ] (Only a suggestion) Hover over card with the "full description" of the docs for each metric.
- The Unit page where there is: `All tests that have been performed on "..." are listed here.`
    - [ ] The list points to Test session, not "tests". Change this in the subtitle.
    - [ ] Convert the User ID to user name or email.
    - [ ] User column title -> Operator.
    - [ ] Formatting the date? Could be in another issue, but it would looks 100000x better :)
- Test Result Metrics:
    - [ ] Add generated data for the metrics (so the graph is not empty when someone is looking at the demo).
    - [ ] The page "test/test_id" doesn't have a breadcrumb. Not sure what should be the order in it though.
- Station Page:
    - [ ] Let's remove the "Global Statistics" placeholder.
- [ ] Let's convert "Production Line" to "Test Profile" in the UI. That slight change will help for the demo a lot (since they are using this term everywhere).
    - I'll do the same in the Sequencer

**Review by LatentDream — APPROVED** [review]
**Feature wise**: All requirements are there :)

**Review by itsjoeoui — APPROVED** [review]
lgtm

File: apps/server/migrations/0010_session.ts [hunk 1]
```diff
@@ -15,6 +16,7 @@ export async function up(db: Kysely<unknown>): Promise<void> {
     .addColumn("station_id", "text", (col) =>
       col.notNull().references("station.id").onDelete("cascade"),
     )
+    .addColumn("duration_ms", "integer", (col) => col.notNull())
```
Comment (itsjoeoui) [inline apps/server/migrations/0010_session.ts:19]: could you add the schema change to a separate file and use `alterTable`. This way we no longer drop the tables.

**Review by itsjoeoui — CHANGES_REQUESTED** [review]

**Review by itsjoeoui — APPROVED** [review]
lgtm

**Pull request merged**